lib/sensors/mux.hpp

```diff
@@ -10,6 +10,7 @@
 namespace hyped::sensors {
 
 static constexpr std::uint8_t kDefaultMuxAddress = 0x70;
+static constexpr std::uint8_t kMaxChannelNumber  = 8;
```

Comment:
To me `kMaxChannelNumber` should be `7` (based on the name)

However, I agree with the choice of 8. Rename to `kMaxNumChannels`?